app/components/forms/ShippingForm/index.js

```diff
@@ -73,8 +74,8 @@ const validate = (vals) => {
 };
 
 ShippingForm.propTypes = {
-  handleSubmit: PropTypes.func,
-  handleCancel: PropTypes.func,
+  // handleSubmit: PropTypes.func,
```

Comment:
Just a reminder for myself, I was working on this last night and need to get this into a working state.